src/Microsoft.WSMan.Management/CredSSP.cs

```diff
@@ -126,7 +126,7 @@ private void DisableClientSideSettings()
                 XmlNamespaceManager nsmgr = new XmlNamespaceManager(resultopxml.NameTable);
                 nsmgr.AddNamespace("cfg", helper.CredSSP_XMLNmsp);
                 XmlNode xNode = resultopxml.SelectSingleNode(helper.CredSSP_SNode, nsmgr);
-                if (!(xNode == null))
+                if (xNode != null)
```

Comment:
Since it is small PR we could use `is null` and `is not null` here and below.

Reply:
See 4131435